### DISCUSSION TITLE
srsRAN to accept RC style 3?

### DISCUSSION THREAD

**Discussion by Keilwerth11270** [srsran/srsRAN_Project#968] [Ideas & Suggestions]
According to the [srsRAN documentation](https://docs.srsran.com/projects/project/en/latest/tutorials/source/near-rt-ric/source/index.html), when it comes to support for RAN Control messages:

```
E2SM_RC service model:
    Only Control Service Style 2 is supported
```

Despite having support for intra-gNB handover since release [24.4](https://docs.srsran.com/projects/project/en/latest/general/source/5_release_notes.html), there is no documentation that srsRAN supports style 3 for RC messages from the E2 interface.

Does srsRAN support inter-cell/intra-gNB handover via style 3 RC messaging and/or are there plans to implement this in the future? 

Handover can [already be initiated](https://docs.srsran.com/projects/project/en/latest/tutorials/source/handover/source/index.html) manually by inputting commands during `gnb` runtime. So if we have that working with E2 messages it would be really awesome!!

**Comment by pgawlowicz** [comment]
Yes, it is on our todo list.

**Reply by Keilwerth11270** [reply]
Is there any roadmap for this feature/approximate release date?
